src/main/kotlin/org/rust/ide/inspections/RsLint.kt

```diff
@@ -33,7 +37,12 @@ enum class RsLint(
         .flatMap { it.queryAttributes.metaItems }
         .filter { it.metaItemArgs?.metaItemList.orEmpty().any { it.id == id || it.id in groupIds } }
         .mapNotNull { it.name?.let { RsLintLevel.valueForId(it) } }
-        .firstOrNull()
+        .let {
```

Comment:
Do we really need it?

Reply:
If implemented using the above method, I think we will no longer need it.

Reply:
Yes i think we still need it to avoid the `forbid` declaration at the superior level.

But let me change the code a little bit and return `FORBID` instead of `DENY`.